geomstats/geometry/stratified/point_set.py

```diff
@@ -3,44 +3,50 @@
 Lead authors: Anna Calissano & Jonas Lueg
 """
 
+import functools
 from abc import ABC, abstractmethod
-from typing import TypeVar
 
 
-def belongs_vectorize(fun):
-    r"""Vectorize the belongs acting on Point as lists."""
+def _vectorize_point(*positions):
```

Comment:
Sorry I missed this one before: is `positions` different than `points`? It seems to me that they refer to the same concept. If this is the case, they should have the same name: consistent names really helps users.

Reply:
This is a decorator that accepts keyword arguments. In order to use it, the user should specify the arg position and names. Check l105 here (https://github.com/annacalissano/geomstats/blob/spider/geomstats/geometry/stratified/spider.py). 

Maybe the name name should be adapted, I couldn't come up with better ideas (args_positions?).

The idea with this decorator is that we can address very different input orders and names with a very robust decorator.